githubteacher/ichenhausen.md

```diff
@@ -0,0 +1 @@
+# Ichenhausen
```

Comment:
Would be great if you could add some places to eat 🍖  and some fun places 🎡, too